lib/amazon-pricing/ec2-dhi-price-list.rb

```diff
@@ -40,6 +40,7 @@ def os_types
       'm5d' => { "large"=>48, "xlarge"=>24, "2xlarge"=>12, "4xlarge"=>6, "12xlarge"=>2, "24xlarge"=>1 },
       'i2' => { "xlarge"=>8, "2xlarge"=>4, "4xlarge"=>2, "8xlarge"=>1, "16xlarge"=>1 },
       'i3' => { "large"=>32, "xlarge"=>16, "2xlarge"=>8, "4xlarge"=>4, "8xlarge"=>2, "16xlarge"=>1, "metal"=>1 },
+      'u' => { "6tb1.metal"=>3 , "9tb1.metal"=> 2, "12tb1.metal" => 1 },
```

Comment:
this seems (potentially) dangerous, to assume the family name is 'u', and further that the size is multi-part (with embedded '.').